src/components/com_tjdashboard/site/tjdashboard.php

```diff
@@ -18,6 +18,26 @@
 $script  = 'const root_url = "' . Juri::root() . '";';
 $document->addScriptDeclaration($script, 'text/javascript');
 
+// Get Tjdashboard params
+$tjdashboardparams = JComponentHelper::getParams('com_tjdashboard');
+
+if ($tjdashboardparams->get('load_bootstrap') == 1)
+{
+	// Load bootstrap CSS and JS.
+	JHtml::stylesheet('media/techjoomla_strapper/bs3/css/bootstrap.css');
+
+	JHtml::_('bootstrap.framework');
+}
+
+if (JVERSION < '3.0')
+{
+	define('COM_TJDASHBOARD_WRAPPER_DIV', 'techjoomla-bootstrap tjlms-wrapper');
```

Comment:
what is `tjlms` used in `tjdashboard` for?

Reply:
removed